eng/ci/code-mirror.yml

```diff
@@ -4,6 +4,7 @@ trigger:
     # Keep this set limited as appropriate (don't mirror individual user branches).
     - main
     - release/*
+    - feature/net10
```

Comment:
Do we want to just add `feature/*`?

Reply:
May be we should make that wild card in the main branch?